src/main/java/org/kitteh/irc/client/library/Client.java

```diff
@@ -180,6 +181,16 @@
         Builder realName(@Nonnull String name);
 
         /**
+         * You can tell KICL to include a message in certain events with this option.
+         *
+         * @param defaultInteractions A class that implements InteractionMap in order to tell KICL about some
+         *                            default values on certain events and interactions.
```

Comment:
Doesn't match commenting style. Limit 78 columns, restart all the way back at `* ` not indented to match above line.